src/pybind/mgr/cephadm/utils.py

```diff
@@ -23,7 +23,8 @@ class CephadmNoImage(Enum):
 # NOTE: order important here as these are used for upgrade order
 CEPH_TYPES = ['mgr', 'mon', 'crash', 'osd', 'mds', 'rgw', 'rbd-mirror', 'cephfs-mirror']
 GATEWAY_TYPES = ['iscsi', 'nfs']
```

Comment:
Perhaps in a separate patch, do we always want to move iscsi and nfs back into the CEPH_TYPES?  So that their ``ceph.conf``s are updated?

Reply:
I think that can be done at the same time we fix the reconfigs during upgrade. That's why they got their own group to begin with.